lib/Pipfile

```diff
@@ -18,7 +18,6 @@ sphinx-rtd-theme = "*"
 recommonmark = "*"
 # Packages used in unit tests:
 matplotlib = "*"
-scipy = "<1.3.0"
```

Comment:
Does this belong to this diff?

Reply:
There was some issue with `scipy` at some point. I guess that's why it's there.